enterprise/api/v1/serializers.py

```diff
@@ -840,7 +840,17 @@ def validate(self, data):  # pylint: disable=arguments-renamed
         if error:
             raise serializers.ValidationError(error)
 
+        uuid = data.get('uuid')
         delivery_method = data.get('delivery_method')
+        if uuid:
+            delivery_method_error = EnterpriseCustomerReportingConfiguration.validate_delivery_method(
```

Comment:
validate_delivery_method is not a class method or static method, how can you call without its object?